code/modules/cm_preds/yaut_bracers.dm

```diff
@@ -406,6 +406,8 @@
 		var/atom/loc = get_true_location(I)
 		if(is_honorable_carrier(recursive_holder_check(I)))
 			continue
+		if(I.anchored)
```

Comment:
this should be above the honorable carrier check because it'll make the processing a bit faster